istioctl/cmd/istioctl/convert/testdata/v1alpha3/egress-rule-tcp-wikipedia-cidr.yaml.golden

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: default
 spec:
   addresses:
-  - 91.198.174.192/27
+  - ip: 91.198.174.192/27
```

Comment:
I don't think adding ip: prefix is an option - this kind of names are in wide use.

How about using a prefix for unix or other kind of addresses ( uds:foo )